app/controllers/add-preprint.js

```diff
@@ -9,7 +10,43 @@ export const State = Object.freeze(new Ember.Object({
     EXISTING: 'existing'
 }));
 
-export default Ember.Controller.extend(NodeActionsMixin, {
+// preprint form basic validations
+const Validations = buildValidations({
+    title: {
+        description: 'Title',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                // minimum length for title?
+                max: 200,
+            })
+        ]
+    },
+    abstract: {
+        description: 'Abstract',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                // currently min of 20 characters -- this is what arXiv has as the minimum length of an abstract
```

Comment:
Nice research of similar products in the space- benchmarking similar services will help us build something tailored for preprints. Thanks for checking!